niap-cc/Permissions/Companion/build.gradle

```diff
@@ -38,3 +39,98 @@ allprojects {
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
+
+import static com.google.common.base.CaseFormat.*
+
+String.metaClass.caseFormat = { from, to ->
+    from.to( to, delegate )
+}
+
+//CodeGenerator
+class Android13BindPlugin implements Plugin<Project> {
```

Comment:
This looks great! So for future releases we can update this with the new BIND_ permissions introduced in the release and use this to generate both the new service classes and manifest tags?

Reply:
yes. also I prepared a trix and gscript to find  differencies of AndroidManifest.xml between android versions.

Reply:
Thanks for adding all this!